src/java.base/windows/classes/sun/nio/ch/FileKey.java

```diff
@@ -45,24 +45,21 @@ public static FileKey create(FileDescriptor fd) throws IOException {
         return fk;
     }
 
+    @Override
     public int hashCode() {
         return (int)(dwVolumeSerialNumber ^ (dwVolumeSerialNumber >>> 32)) +
                (int)(nFileIndexHigh ^ (nFileIndexHigh >>> 32)) +
-               (int)(nFileIndexLow ^ (nFileIndexHigh >>> 32));
+               (int)(nFileIndexLow ^ (nFileIndexLow >>> 32));
```

Comment:
Well spotted, probably a cut and paste error at some point, but benign, the resulting hash code is still okay.